extensions/amp-consent/0.1/consent-ui.js

```diff
@@ -74,6 +75,11 @@ export class ConsentUI {
     /** @private {?Element} */
     this.ui_ = null;
 
+    /** @private {boolean} */
+    this.overlayEnabled_ = isExperimentOn(baseInstance.win, 'amp-consent-v2') &&
+      config['uiConfig'] &&
+      !!config['uiConfig']['overlay'];
```

Comment:
nit: can we do `config['uiConfig']['overlay'] === true`, this is to prevent `'overlay': 'false'` case.